exercises/concept/squeaky-clean/.meta/Exemplar.elm

```diff
@@ -0,0 +1,45 @@
+module SqueakyClean exposing (clean1, clean2, clean3, clean4, clean5)
+
+
+clean1 : String -> String
+clean1 =
+    String.replace " " "_"
+
+
+clean2 : String -> String
+clean2 str =
+    List.foldl (\ctrl -> String.replace ctrl "[CTRL]") (clean1 str) [ "\n", "\u{000D}", "\t" ]
+
+
+clean3 : String -> String
+clean3 str =
+    case String.split "-" (clean2 str) of
+        [] ->
+            ""
+
+        first :: rest ->
+            String.concat (first :: List.map capitalize rest)
+
+
+capitalize : String -> String
+capitalize str =
+    case String.uncons str of
+        Nothing ->
+            str
+
+        Just ( first, rest ) ->
+            String.fromChar (Char.toUpper first) ++ rest
+
+
+clean4 : String -> String
+clean4 =
+    clean3 >> String.filter (Char.isDigit >> not)
+
+
+clean5 : String -> String
+clean5 =
```

Comment:
In this instructions, you left the name `clean` instead of `clean5`. Actually that might be a good idea for the last function to be called `clean`.